modules/appeals_api/app/services/appeals_api/pdf_construction/supplemental_claim/v3/form_fields.rb

```diff
@@ -0,0 +1,74 @@
+            veteran_middle_initial: 'form1[0].#subform[2].VeteransMiddleInitial1[0]',
+            veteran_ssn_first_three: 'form1[0].#subform[2].SocialSecurityNumber_FirstThreeNumbers[0]',
+            veteran_ssn_middle_two: 'form1[0].#subform[2].SocialSecurityNumber_SecondTwoNumbers[0]',
+            veteran_ssn_last_four: 'form1[0].#subform[2].SocialSecurityNumber_LastFourNumbers[0]',
+            veteran_file_number: 'form1[0].#subform[2].VAFileNumber[0]',
+            veteran_dob_day: 'form1[0].#subform[2].DOBday[0]',
+            veteran_dob_month: 'form1[0].#subform[2].DOBmonth[0]',
+            veteran_dob_year: 'form1[0].#subform[2].DOByear[0]',
+            veteran_service_number: 'form1[0].#subform[2].VeteransServiceNumber[0]',
+            veteran_insurance_policy_number: 'form1[0].#subform[2].InsurancePolicyNumber[0]',
+            claimant_middle_initial: 'form1[0].#subform[2].ClaimantsMiddleInitial1[0]',
+            signing_appellant_state_code: 'form1[0].#subform[2].CurrentMailingAddress_StateOrProvince[0]',
+            signing_appellant_country_code: 'form1[0].#subform[2].CurrentMailingAddress_Country[0]',
+            international_phone: 'form1[0].#subform[2].TELEPHONE[0]',
+            phone_area_code: 'form1[0].#subform[2].Daytime1[0]',
+            phone_prefix: 'form1[0].#subform[2].Daytime2[0]',
+            phone_line_number: 'form1[0].#subform[2].Daytime3[0]',
+            claimant_type_code: 'form1[0].#subform[2].RadioButtonList[1]',
+            benefit_type_code: 'form1[0].#subform[2].RadioButtonList[0]',
+            form_5103_notice_acknowledged: 'form1[0].#subform[3].TIME1230TO2PM[0]',
+            veteran_claimant_rep_date_signed: 'form1[0].#subform[3].DATESIGNED[0]',
+            alternate_signer_date_signed: 'form1[0].#subform[3].DATESIGNED[1]'
+          }.freeze
+
+          def initialize
+            FIELD_NAMES.each { |field, name| define_singleton_method(field) { name } }
+          end
+
+          # rubocop:disable Metrics/MethodLength
+          def boxes
+            {
+              veteran_first_name: { at: [-1, 601.6], width: 201, height: 13.9 },
+              veteran_last_name: { at: [235.5, 601.5], width: 304, height: 15.3 },
+              claimant_first_name: { at: [-1, 496.5], width: 200, height: 14.4 },
+              claimant_last_name: { at: [236, 497.25], width: 302, height: 14.7 },
+              claimant_type_other_text: { at: [434, 470], width: 105 },
+              signing_appellant_number_and_street: { at: [21.4, 436.5], width: 516, height: 13.2 },
+              signing_appellant_city: { at: [193.3, 415.3], width: 309, height: 13.7 },
+              signing_appellant_zip_code: { at: [290.7, 393.4], width: 82, height: 14.7 },
+              international_phone: { at: [324, 361], width: 200, height: 14 },
+              signing_appellant_email: { at: [-1, 333.7], width: 524, height: 12.9 },
+              contestable_issues: Structure::MAX_ISSUES_ON_MAIN_FORM.times.map do |i|
```

Comment:
The only reason this isn't failing tests by referring to the `Strucure` is that I don't have tests for `boxes` - this value will come in my next PR - see how that will look in my [original PR here](https://github.com/department-of-veterans-affairs/vets-api/pull/11863/files#diff-7ff4ea3dfbd43101051cf51cb112bf658ff1502f2f05c17f59d38fc9fbf9cf01R8), which was closed for being too long